docs/snapshotters/erofs.md

```diff
@@ -128,6 +128,36 @@ $ # run the container with the provides snapshotter
 $ ctr run -rm -t --snapshotter erofs docker.io/library/busybox:latest hello sh
 ```
 
+## Data Integrity
+
+The EROFS snapshotter provides two methods to consolidate data integrity:
+
+### Data Integrity with Immutable File Attribute
+
+By setting `set_immutable = true`, the EROFS snapshotter marks each layer blob
+with `IMMUTABLE_FL`. This ensures that dirty data is flushed immediately and the
+EROFS layer blob cannot be deleted, renamed, or modified.
+
+The immutable file attribute is mainly used to ensure data persistence and
+prevent artificial data loss, but it cannot detect data corruption caused by
+hardware failures. Since it can flush in-memory dirty data, it may significantly
+increase the time it takes to launch a container: for example, the unpacking time
```

Comment:
Sounds like this really means increase time to pull+unpack an image?

Reply:
yes, accurately speaking, it increases time to unpack an image.